src/ucs/datastruct/ptr_array.c

```diff
@@ -136,17 +135,18 @@ void ucs_ptr_array_cleanup(ucs_ptr_array_t *ptr_array)
     }
 
     ucs_free(ptr_array->start);
+    ucs_free((void*)ptr_array->name);
     ucs_ptr_array_clear(ptr_array);
 }
 
-static void ucs_ptr_array_grow(ucs_ptr_array_t *ptr_array, unsigned new_size
-                               UCS_MEMTRACK_ARG)
+static void ucs_ptr_array_grow(ucs_ptr_array_t *ptr_array, unsigned new_size,
+                               const char* alloc_name)
```

Comment:
maybe not pass alloc_name. take it from ptr_array->name

Reply:
> maybe not pass alloc_name. take it from ptr_array->name

done